dd-sdk-android-core/src/test/kotlin/com/datadog/android/v2/core/DatadogCoreTest.kt

```diff
@@ -299,13 +300,14 @@ internal class DatadogCoreTest {
     ) {
         // Given
         val fakeReceiver = FeatureEventReceiver { }
+        reset(mockInternalLogger)
```

Comment:
why do we need to reset it?

Reply:
There was some flakiness because some steps during the setup could send logs to the internal logger